src/_locales/en/messages.json

```diff
@@ -190,6 +186,98 @@
   "footnoteUnsupportedOSMessage": {
     "message": "We’re working to bring support to macOS and Linux soon!"
   },
+  "turnOnProtection": {
+    "message": "Turn on protection",
+    "description": "Button label"
+  },
+  "removeCustomLocation": {
+    "message": "Remove custom location",
+    "description": "Button label"
+  },
+  "websiteProtection": {
+    "message": "Website protection",
+    "description": "A subtitle that is followed by an off icon and the word 'Off'"
+  },
+  "giveFeedback": {
+    "message": "Give feedback",
+    "description": "Button label"
+  },
+  "learnMore": {
+    "message": "Learn more",
+    "description": "Label of link that leads to SUMO"
+  },
+  "onboarding1_title": {
+    "message": "Turn off VPN for specific websites",
+    "description": "Title in onboarding panel"
+  },
+  "onboarding1_body": {
+    "message": "With the Firefox extension, you can turn off VPN protection for specific websites, while keeping it on for everything else.",
+    "description": "Body text in onboarding panel"
+  },
+  "onboarding2_title": {
+    "message": "See different websites from different locations",
+    "description": "Title in onboarding panel"
+  },
+  "onboarding2_body": {
+    "message": "",
```

Comment:
@lesleyjanenorton 
This string is empty.

I think it should be `News from London. Sports from Sao Paulo. Shopping from Tokyo. Set website-specific locations for personalized browsing.`